ansible/roles/blackbox_exporter/templates/blackbox_exporter.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Prometheus Blackbox Exporter
+
+[Service]
+User=nobody
+Group=nogroup
+EnvironmentFile=-/etc/default/blackbox_exporter
+PermissionsStartOnly=true
+ExecStartPre=/sbin/setcap 'cap_net_raw+ep' {{ blackbox_exporter_path }}/blackbox_exporter
```

Comment:
According to http://man7.org/linux/man-pages/man7/capabilities.7.html, there seems to be no other way to grant `SOCK_DGRAM`, `IPPROTO_ICMP`.